rust/src/daemon.rs

```diff
@@ -108,6 +108,13 @@ fn deployment_populate_variant_origin(
         "requested-base-local-replacements",
         tf.derive.override_replace_local.as_ref(),
     );
+    if let Some(remote_overrides) = tf.derive.override_replace.as_ref() {
+        let v: Vec<(String, Vec<&String>)> = remote_overrides
+            .iter()
+            .map(|ovr| (ovr.from.to_string(), ovr.packages.iter().collect()))
+            .collect();
+        dict.insert_value("requested-base-remote-replacements", &v.to_variant());
```

Comment:
I'm OK with this but I'm wondering...can we instead try to use this as a cutover point where we have the client directly read the treefile?   (And I guess `rpm-ostree status --json` would include the json serialization of the override treefile?)